osf/models/node.py

```diff
@@ -2428,16 +2429,12 @@ def _remove_from_associated_collections(self, auth=None, force=False):
                     comment='Request to review this submission was canceled due to privacy changes.',
                 )
             elif force and submission.state == CollectionSubmissionStates.ACCEPTED:
-                request, user_id = get_request_and_user_id()
```

Comment:
This was a fix for a problem I had when using the admin to spam something, when it would flag a pending submission a user object wouldn't be passed (because it was via an admin which doesn't have access to the request context), so it would fail, the fix was just to allow it to be None.